### ISSUE TITLE
No hot reloading on JS code changes

### ISSUE THREAD

**Issue by ShaiBer** [winglang/wing#5949]
### I tried this:

Write some wing code with extern in JS, make changes to the JS code

### This happened:

Nothing

### I expected this:

See the changes reflected in the console with hot reloading, same as with the wing code

### Is there a workaround?

kill the console and run wing it again

### Wing Version

0.61.1

### Platform(s)

MacOS

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by ainvoner** [comment]
I guess having [this](https://github.com/winglang/wing/issues/3730) would be amazing for this one as well...

**Comment by skyrpex** [comment]
Yeah, there's hardly anything we can do without https://github.com/winglang/wing/issues/3730.

When it gets implemented in the compiler, we can hook to it and reload as necessary.
